app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -247,13 +248,16 @@ export default Component.extend(ComposerUploadUppy, {
       });
     }
 
-    this.element
-      .querySelector(".d-editor-input")
-      ?.addEventListener("scroll", this._throttledSyncEditorAndPreviewScroll);
+    input?.addEventListener(
+      "scroll",
+      this._throttledSyncEditorAndPreviewScroll
+    );
+
+    this._registerImageAltTextButtonClick(preview);
```

Comment:
This was being called in the `previewUpdated` method which was called every time the preview was, well, updated 😬 

We don't need to register those events every time the preview is updated. Instead, we only register when the composer is opened.